SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -2127,15 +2227,15 @@ void CFEASolver::BC_Damper(CGeometry *geometry, CNumerics *numerics, CConfig *co
 
 }
 
-void CFEASolver::BC_Deforming(CGeometry *geometry, CNumerics *numerics, CConfig *config, unsigned short val_marker){
+void CFEASolver::BC_Deforming(CGeometry *geometry, CNumerics *numerics, const CConfig *config, unsigned short val_marker){
 
   for (auto iVertex = 0ul; iVertex < geometry->nVertex[val_marker]; iVertex++) {
 
     /*--- Get node index ---*/
     auto iNode = geometry->vertex[val_marker][iVertex]->GetNode();
 
     /*--- Retrieve the boundary displacement ---*/
-    su2double Disp[3] = {0.0, 0.0, 0.0};
+    su2double Disp[3] = {0.0};
```

Comment:
Use MAXNDIM instead of the 3's throughout?

Reply:
Yeap, I've been delaying that out of laziness.